pkg/framework/aries/api/vdr/vdr.go

```diff
@@ -24,17 +23,96 @@ const DIDCommServiceType = "did-communication"
 
 // Registry vdr registry.
 type Registry interface {
-	Resolve(did string, opts ...resolve.Option) (*did.DocResolution, error)
+	Resolve(did string, opts ...ResolveOption) (*did.DocResolution, error)
 	Store(doc *did.Doc) error
-	Create(method string, opts ...create.Option) (*did.DocResolution, error)
+	Create(method string, did *did.Doc, opts ...CreateOption) (*did.DocResolution, error)
 	Close() error
 }
 
 // VDR verifiable data registry interface.
 type VDR interface {
-	Read(did string, opts ...resolve.Option) (*did.DocResolution, error)
-	Store(doc *did.Doc, by *[]doc.ModifiedBy) error
-	Build(keyManager kms.KeyManager, opts ...create.Option) (*did.DocResolution, error)
+	Read(did string, opts ...ResolveOption) (*did.DocResolution, error)
+	Store(doc *did.Doc, by *[]ModifiedBy) error
+	Build(keyManager kms.KeyManager, did *did.Doc, opts ...CreateOption) (*did.DocResolution, error)
```

Comment:
We should have a TODO to rename this function.

Reply:
Ok

Reply:
Should I rename it to Create

Reply:
yes - suggest matching the function names in did-core.

Reply:
I think the CreateOptions go with the `Build` function not the `Create` function.

Reply:
I removed CreateOptions